### PR TITLE
fix: update when not to use Accelerate

### LINKED ISSUE #5295: update faq regarding when Accelerate should not be used
This [section](https://www.prisma.io/docs/data-platform/accelerate/faq#when-should-i-not-use-accelerate) of the docs explains why Accelerate may not be suitable for users. However, users would still benefit from Accelerate's connection pool; the section must be updated to reflect that.

### PR DESCRIPTION
**Pull request by ankur-arch**
closes #5295

### CODE REVIEW

File: content/800-data-platform/100-accelerate/600-faq.mdx [hunk 1]
```diff
@@ -86,14 +84,20 @@ The freshness of the data served by the cache depends on the cache strategy defi
...
 - Your application data _always_ needs to be up-to-date on retrieval, which would be difficult to set a reasonable cache strategy.
 
+Even without using Accelerate's global cache, you can still greatly benefit from Accelerate by using its connection pool, especially in serverless or edge functions, where it is difficult to manage and scale database connections. You can learn more about the serverless challenge [here](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#the-serverless-challenge).
+
 ## Can I use Accelerate with other ORMs/query builders/drivers?
 
 No. We currently do not have any plans for supporting other ORMs/query builders or drivers. However, if you're interested in support for other libraries, feel free to [reach out and let us know](https://prisma.slack.com/archives/C04KTN0V2Q0).
```
Comment (petradonka) [inline content/800-data-platform/100-accelerate/600-faq.mdx]: Can we replace the occurrences of Slack here with Discord?

**Review by petradonka — APPROVED** [review]
Thanks Ankur! I'll let others take a review also.

File: content/800-data-platform/100-accelerate/600-faq.mdx (hunk 1 above)
Comment (ankur-arch) [inline content/800-data-platform/100-accelerate/600-faq.mdx]:
```suggestion
No. We currently do not have any plans for supporting other ORMs/query builders or drivers. However, if you're interested in support for other libraries, feel free to [reach out and let us know](https://pris.ly/discord).
```

File: content/800-data-platform/100-accelerate/600-faq.mdx [hunk 2]
```diff
@@ -86,14 +84,20 @@ The freshness of the data served by the cache depends on the cache strategy defi
...
 - Your application data _always_ needs to be up-to-date on retrieval, which would be difficult to set a reasonable cache strategy.
 
+Even without using Accelerate's global cache, you can still greatly benefit from Accelerate by using its connection pool, especially in serverless or edge functions, where it is difficult to manage and scale database connections. You can learn more about the serverless challenge [here](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#the-serverless-challenge).
+
 ## Can I use Accelerate with other ORMs/query builders/drivers?
 
-No. We currently do not have any plans for supporting other ORMs/query builders or drivers. However, if you're interested in support for other libraries, feel free to [reach out and let us know](https://prisma.slack.com/archives/C04KTN0V2Q0).
+No. We currently do not have any plans for supporting other ORMs/query builders or drivers. However, if you're interested in support for other libraries, feel free to [reach out and let us know](https://pris.ly/discord).
```
Comment (ankur-arch) [inline content/800-data-platform/100-accelerate/600-faq.mdx]:
```suggestion
No. We currently do not have any plans for supporting other ORMs/query builders or drivers. However, if you're interested in support for other libraries, feel free to reach out and let us know in our [Discord](https://pris.ly/discord) community in the `#accelerate-feedback` channel.
```

**Review by nikolasburk — APPROVED** [review]

**Comment by nikolasburk** [comment]
looks good, thanks Ankur 🚀

**Pull request merged**